src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -517,7 +554,34 @@ const MeasureGroups = () => {
                   <MenuItem
                     data-testid="stratifications-tab"
                     isActive={activeTab == "stratification"}
-                    onClick={() => setActiveTab("stratification")}
+                    onClick={() => {
+                      setActiveTab("stratification");
+                      if (!!formik.values.stratifications) {
+                        while (formik.values.stratifications.length < 2) {
+                          formik.values.stratifications.push({
+                            cqlDefinition: "",
+                            description: "",
+                            association: "",
+                            id: "",
+                          });
+                        }
+                      } else {
+                        formik.values.stratifications = [
+                          {
```

Comment:
seems like we can avoid duplication of this object at multiple places. we can define this object at one place and use the spread operator to copy wherever needed

Reply:
Fixed, thank you